src/sidebar/components/test/new-note-test.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var angular = require('angular');
+
+var util = require('../../directive/test/util');
+
+describe('newNote', function () {
+  before(function () {
+    angular.module('app', [])
+      .component('selectionTabs', require('../selection-tabs'))
+      .component('newNote', require('../new-note'));
+  });
+
+  beforeEach(function () {
+    var fakeAnnotationUI = {};
+    var fakeFeatures = {
+      flagEnabled: sinon.stub().returns(true),
+    };
+    var fakeSettings = { theme: 'clean' };
+
+    angular.mock.module('app', {
+      annotationUI: fakeAnnotationUI,
+      features: fakeFeatures,
+      settings: fakeSettings,
+    });
+  });
+
+  it('should display the new note button when the notes tab is active', function () {
+    var elem = util.createDirective(document, 'selectionTabs', {
```

Comment:
It looks to me like these functions are testing behavior of the `selectionTabs` component, not the `newNote` component, so they should be added to the existing tests for that component.